Header header

float64 wind_speed # Speed of wind in m/s
float64 wind_angle # Angle of wind in degrees
